weatherman.py

```diff
@@ -1,4 +1,58 @@
 """
 weatherman is a software that generates reports about the past weather of murree
 """
-# this file will contain code in the future
+import re
+import sys
+
+from modules.extreme_year import extreme_temperatures_year as t1
+
+
+def main():
+    """
+    this is a driver function for weatherman.
+    :return:
+    """
+    parameters = sys.argv[1:]
+    accepted_flags = ["-e"]
+    accepted_regex = [r"\d{4}"]
+    flag_handler = [t1]
+    i = 0
+    path = ""
```

Comment:
it does not remain constant. the variable path can change if the user includes it in command line parameters. if u still want me to define it outside main, let me know.

Reply:
Even in that case, it would be a good idea to keep the path as a constant value in something like `DEFAULT_FILES_DIR=''`. You can create a `constants.py` file and put it there. It would serve the purpose for all the instances in which the user would not supply a path value.

Reply:
done